rtos/TARGET_CORTEX/mbed_boot.c

```diff
@@ -320,6 +320,12 @@ void mbed_start_main(void)
     _main_thread_attr.cb_mem = &_main_obj;
     _main_thread_attr.priority = osPriorityNormal;
     _main_thread_attr.name = "main_thread";
+
+    /* Allow main thread to call secure functions */
+#if (__DOMAIN_NS == 1U)
```

Comment:
Is this always defined (shouldn't this check first if its defined then its value) ?

Reply:
Yes it is always defined in https://github.com/ARMmbed/mbed-os/blob/eefb84c0dbd2fe0565fe2dd7888d83775f00cc12/rtos/TARGET_CORTEX/rtx5/core_cm.h#L64